transports/janus_http.c

```diff
@@ -117,6 +117,7 @@ static janus_transport_callbacks *gateway = NULL;
 static gboolean http_janus_api_enabled = FALSE;
 static gboolean http_admin_api_enabled = FALSE;
 static gboolean notify_events = TRUE;
+static enum MHD_FLAG mhd_debug_flag = MHD_NO_FLAG;
```

Comment:
Hve you checked if this breaks older versions of libmicrohttpd? We had to do something like this for `enum MHD_result` recently.

Reply:
It was `enum MHD_FLAG` in MHD version 0.9.59, so unless they changed it in an intervening version and then changed it back we should be ok